spec/unit/libraries/recipe_spec.rb

```diff
@@ -47,4 +48,93 @@
       end
     end
   end
+
+  describe '.my_cluster_data' do
+    before do
+      allow(ChefVault::Item).to receive(:data_bag_item_type).and_return(:vault)
+      allow(ChefVault::Item).to receive(:load).with('cerner_splunk', 'cluster_bag').and_return(cluster_configs)
+      allow(ChefVault::Item).to receive(:load).with('cerner_splunk', 'multisite_bag').and_return(multisite_configs)
+    end
+
+    after do
+      CernerSplunk.reset
+    end
+
+    let(:clusters) { ['cerner_splunk/cluster_bag'] }
+    let(:node) { { 'splunk' => { 'config' => { 'clusters' => clusters } } } }
+    let(:cluster_configs) { { 'site' => 'site1', 'multisite' => 'cerner_splunk/multisite_bag', 'apps' => 'cerner_splunk/apps', 'indexes' => 'cerner_splunk/overriden-indexes' } }
```

Comment:
minor, overridden should have two d's

Reply:
0cd73923d94c8e5f81dede79457fbf21d959f065